line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingService.java

```diff
@@ -329,6 +330,12 @@
     @POST("v2/bot/user/{userId}/linkToken")
     Call<IssueLinkTokenResponse> issueLinkToken(@Path("userId") String userId);
 
+    /**
+     * Retrieves the demographic attributes for a bot's friends.
```

Comment:
```suggestion
     * Gets the demographic attributes for a bot's friends.
```

Reply:
Sorry, official document write so. It's ok.

![image](https://user-images.githubusercontent.com/1370068/65846801-fd327080-e379-11e9-82b4-2d21f4829e22.png)